stdlib/test/collections/test_list.mojo

```diff
@@ -348,6 +348,69 @@ def test_list_insert():
     for i in range(len(v4)):
         assert_equal(v4[i], i + 1)
 
+def test_list_index():
+    var test_list_a = List[Int](10,20,30,40,50)
+    
+    # Basic Functionality Tests
+    assert_equal(__type_of(test_list_a).index(test_list_a, 10), 0)  # Verify finding the first element's index.
```

Comment:
I think the comments are a bit too much here. Comments should be there to explain something being done implicitely in the code, or to provide context that the reader can't guess, or even explain complexe algorithms. Here the comments are explaining in english what the code does. I don't think they're useful. The comments separating each section are relevant though.

Reply:
Thanks. Done. It's now definitely cleaner.